main/main.c

```diff
@@ -45,6 +47,22 @@ uint64_t cycles_per_sec;
 uint64_t cycles_per_ms;
 uint64_t picosec_per_cycle;
 
+FILE *log_file;
+
+static int
+gatekeeper_log_init(void)
+{
+	log_file = fopen(LOG_FILE_PATH, "w+");
+	if (log_file == NULL) {
```

Comment:
We have not discussed with which user Gatekeeper/Grantor will run. I'm assuming that it must be root to bootstrap, but can we turn to a constrained user after the boot? This is important because depending on which user we are using to run Gatekeeper/Grantor, we may need to change owner and access mode of the log file just created.

I realize that there's no issue open to address the user that Gatekeeper/Grantor will use, so please create an issue for that. This pull request depends on this new issue.